lte/gateway/configs/pipelined.yml

```diff
@@ -123,6 +123,7 @@ qos:
  impl: linux_tc
  enable_pyroute2: true
  max_rate: 1000000000
+ gbr_rate: 80Kbit
```

Comment:
i understand this value is defaulted to 80kbps when gbr is unknown ?

Reply:
right.